lib/solidus_dev_support/templates/extension/bin/setup

```diff
@@ -5,4 +5,4 @@ set -vx
 
 gem install bundler --conservative
 bundle update
-bundle exec rake extension:test_app
+bin/rake clobber
```

Comment:
Same here: there is no `bin/rake` in extensions generated by this gem. Also, will `rake clobber` generate a new test app in addition to removing the old one?

Reply:
It will just remove the old one, the generation is lazy and it's done automatically by the default task if the dummy app is not there by the `directory` rule in the rake task.